modules/common/locals.tf

```diff
@@ -2,4 +2,5 @@ locals {
   vpc_name       = format("atlas-%s-vpc", var.environment_name)
   cluster_name   = format("atlas-%s-cluster", var.environment_name)
   ingress_domain = var.domain != null ? "${var.environment_name}.${var.domain}" : null
+  nat_ip_cidr    = var.whitelist_cidr == ["0.0.0.0/0"] ? [] : formatlist("%s/32", module.vpc.nat_public_ips)
```

Comment:
Can we add unit test for this?

Reply:
Since this value goes into helm_release module, namely into `values` attribute, sadly, it is not available in planned variables for ingress module, so even if I pass a dummy `vpc.nat_public_ips` (i.e. not get it from the module, but pass the module which makes it possible to pass custom values in tests) I can't verify it anywhere.

<img width="822" alt="image" src="https://user-images.githubusercontent.com/52448429/209726680-bc26adfd-f52b-4fd3-be2e-c12be1346629.png">

An alternative is to use `set` in helm release (instead of values) which I explored and updated this PR accordingly. Had to move all the logic to helm module. Not sure it's worth one assertion but it's asserted now :)